deploy/shipyard-all.yaml

```diff
@@ -203,7 +205,7 @@ spec:
         edge/paths: '[{"basePath": "/organizations", "containerPort": "5280"}]'
     spec:
       containers:
-      - image: thirtyx/kiln
+      - image: thirtyx/kiln:0.1.19
```

Comment:
In this deployment file i'd like to leave each component at the latest tag. I know enrober is at v0.6.0 that should be changed.

If you look at the make file there three env vars that can be set for each compent version and the Makefile will pull that version into minikube and tag it as latest.

Not sure if this easier and less error prone then editing the deployment yaml it self it worked well in previous projects because integration testing can be scripted to run multiple versions of each component.